constraints.txt

```diff
@@ -19,3 +19,8 @@ flatbuffers~=1.12.0
 grpcio==1.32.0
 grpcio-health-checking==1.32.0
 six~=1.15.0
+
+torch>=1.7.1,<=1.9.0
+torchvision>=0.8.1,<=0.10.0
```

Comment:
So... you also put torch and torchvision versions to constraints file inside init_venv.sh. Why do you need both?

Reply:
Yes, will remove that

Reply:
Done